src/App.js

```diff
@@ -61,6 +62,7 @@ class App extends Component {
   }
 
   componentDidMount() {
+    Seed();
```

Comment:
Let's not seed the database every time the page loads. Keep the script separate for now

Reply:
Yep, just did it so you can run. It's set to three iterations so not the full thing.